### PR TITLE
Remove sitewide alert for hurricane ian

### PR DESCRIPTION
**Pull request by nick-mon1**
## Summary

Remove Hurricane Ian sitewide alert and put as 2nd item on the "Popular Guides & Resources"

### CODE REVIEW

File: content/resources/hurricane-ian-guidance-for-u-s-government-websites-and-social-media.md [hunk 1]
```diff
@@ -19,7 +19,7 @@ aliases:
 # 0 -- hidden
 # 1 -- visible
 # 2 -- highlighted
-weight: 2
```
Comment (mejiaj) [inline content/resources/hurricane-ian-guidance-for-u-s-government-websites-and-social-media.md:22]: Just curious, is there a limit on the weight? Going by the comment I only see `0, 1, 2`.

File: themes/digital.gov/layouts/partials/core/sitewide-alert.html [hunk 2]
```diff
@@ -1,3 +1,4 @@
+// Creates sitewide banner alert at top of page under the banner
```
Comment (mejiaj) [inline themes/digital.gov/layouts/partials/core/sitewide-alert.html:1]: In the future we should templatize this further by allowing the include to take a component variant, title, and body content. This way we can save some time in the future.

**Review by mejiaj — COMMENTED** [review]
@nick-mon1 looks good! Just some questions. Otherwise LGTM.

File: content/resources/hurricane-ian-guidance-for-u-s-government-websites-and-social-media.md (hunk 1 above)
Comment (nick-mon1) [reply content/resources/hurricane-ian-guidance-for-u-s-government-websites-and-social-media.md:22]: I played around with these values to get the ordering right as requested by Ammie.
Digital.gov convention would be to use `0, 1, 2` but any integer is allowed.

File: themes/digital.gov/layouts/partials/core/sitewide-alert.html (hunk 2 above)
Comment (nick-mon1) [reply themes/digital.gov/layouts/partials/core/sitewide-alert.html:1]: I'll make a note of this and have you show me what you mean by this. I think that would be an excellent idea!

File: content/resources/hurricane-ian-guidance-for-u-s-government-websites-and-social-media.md (hunk 1 above)
Comment (mejiaj) [reply content/resources/hurricane-ian-guidance-for-u-s-government-websites-and-social-media.md:22]: Okay, thanks Nick. I don't think we should stray too far from this guidance. For example, maybe not go higher than a certain weight. Just something to consider for the future.

File: themes/digital.gov/layouts/partials/core/sitewide-alert.html (hunk 2 above)
Comment (mejiaj) [reply themes/digital.gov/layouts/partials/core/sitewide-alert.html:1]: Sure thing, I've created an issue with the feature request
https://github.com/GSA/digitalgov.gov/issues/5514

**Pull request merged**